styles/style.css

```diff
@@ -1,9 +1,114 @@
 * {
   margin: 0;
   padding: 0;
-  box-sizing: 0;
+  box-sizing: border-box;
 }
 
+/*=====testimonial-section====*/
+.testimonials-section {
+  max-width: 1917px;
+  position: relative;
+  overflow: hidden;
```

Comment:
overflow özelliğine ihtiyacın yok, sildiğin zaman bir şey değişmediğini fark edebilirsin